### PR TITLE
Update table_gandi_certificate.go

### PR DESCRIPTION
**Pull request by jdenoy**
Added expiration date of ssl certificate

### CODE REVIEW

**Comment by jdenoy** [comment]
Documentation is automatic for ".inspect gandi_certificate" section.
Reopening PR.

File: gandi/table_gandi_certificate.go [hunk 1]
```diff
@@ -25,6 +25,7 @@ func tableGandiCertificate() *plugin.Table {
 			{Name: "cn", Type: proto.ColumnType_STRING, Transform: transform.FromField("CN"), Description: "Common Name."},
 			{Name: "package_name", Type: proto.ColumnType_STRING, Transform: transform.FromField("Package.Name"), Description: ""},
 			{Name: "status", Type: proto.ColumnType_STRING, Description: "One of: 'pending', 'valid', 'revoked', 'replaced', 'replaced_rev', 'expired'."},
+			{Name: "expiration", Type: proto.ColumnType_STRING, Transform: transform.FromField("Dates.EndsAt"), Description: "Expiration date"},
```
Comment (francois2metz) [inline gandi/table_gandi_certificate.go:28]: The expiration is probably a timestamp. Could you use the TIMESTAMP type for the column please.

Comment (francois2metz) [inline gandi/table_gandi_certificate.go:28]: Please add a dot at the end of the description.

**Review by francois2metz — COMMENTED** [review]
Hi!
Thanks for your contribution. Could you take a look a the comments?

File: gandi/table_gandi_certificate.go (hunk 1 above)
Comment (jdenoy) [reply gandi/table_gandi_certificate.go]: done

**Comment by jdenoy** [comment]
Hello François,

mods requested have been implemented.

**Comment by francois2metz** [comment]
Hi, I merged your commit in 39a96c78a4ab5e96842333028cfd7a126893b9ae. Thanks!

**Pull request closed without merging**